af2slurm-watcher.py

```diff
@@ -1,49 +1,115 @@
 #!python
 from configargparse import ArgParser, ArgumentDefaultsHelpFormatter
+import subprocess
 from time import sleep
 from glob import glob
+import os
+import shutil
 
-def copy_over_fasta_file(file_name: str, in_folder: str, out_folder: str, dry_run: bool = True) -> tuple[str, str]:
-    """Makes a folder in the out_foder with the name of the fasta fie. Copies fasta file from the in folder the new target folder. Returns the path to the fasta file and any first line comments (to be used as arguments for colabfold_batch)"""
-    pass
 
-def create_slurm_submit_line(fast_file, slurm_options, colabfold_options):
+def copy_over_fasta_file(
+    file_name: str, in_folder: str, out_folder: str, dry_run: bool = False
+) -> tuple[str, str]:  # dry_run is seto to 0 which means the comand goes through
+    """Makes a folder in the out_foder with the name of the fasta file. Copies fasta file from the in folder the new target folder. Returns the path to the fasta file and any first line comments (to be used as arguments for colabfold_batch)"""
+    # Check if the file name ends with '.fasta'
```

Comment:
In general too verbose with comments.
It's usually better to comment the general layout of the code, each line does not need comments, especially in python :)